source/nexus.cc

```diff
@@ -109,13 +109,11 @@ G4int main(int argc, char** argv)
 
   // visual mode
   if (!batch) {
-    G4UIExecutive* ui = new G4UIExecutive(argc, argv);
-    G4VisManager* visManager = new G4VisExecutive;
+    std::unique_ptr<G4UIExecutive> ui{new G4UIExecutive{argc, argv}};
+    std::unique_ptr<G4VisManager> visManager{new G4VisExecutive};
```

Comment:
Spot on!

Reply:
We are not using the command-line parameters for whatever use `G4UIExecutive` gives them. Maybe we should pass a 1 as `argc`, as if the user had not provided any argument.